ivy/functional/ivy/general.py

```diff
@@ -1456,6 +1456,184 @@ def value_is_nan(
     ret
         Boolean as to whether the input value is a nan or not.
 
+    Functional Examples
+    --------
+
+    With :code:`ivy.Array` input:
+
+    >>> x = ivy.array([1])
+    >>> y = ivy.value_is_nan(x)
+    >>> print(y)
+    False
+
+    >>> x = ivy.array([1])
```

Comment:
Just to make things neater, you can remove this example and the one after this.

Reply:
Thank you for your comments. Removed .